spec/models/student_spec.rb

```diff
@@ -4,4 +4,18 @@
   describe 'relationships' do
     it { should belong_to :enrollment }
   end
+
+  describe '#list_data' do
+    let!(:eloise_may) { Enrollment.create!(location: 'Eloise May', schedule: DateTime.parse('2030-06-11T15:00:24.000Z'), student_limit: 30) }
```

Comment:
We don't need this line. I forgot it here when I copy pasted the code to save time.